object.c

```diff
@@ -4298,7 +4298,7 @@ InitVM_Object(void)
      *
      * == What's Here
      *
-     * \Module \Kernel provides methods that are useful for:
```

Comment:
This is documentation for the `Kernel` module, so we don't need to unescape it.

Reply:
Fixed.